### PR TITLE
Add utilization bar chart pattern

### PR DESCRIPTION
**Pull request by serenamarie125**
Add the pattern as it currently exists on patternfly.org

### CODE REVIEW

**Comment by andresgalante** [comment]
Thanks a lot @serenamarie125 

@LHinson ready when you are 😄

File: pattern-library/data-visualization/utilization-bar-chart/design/design.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Utilization Bar Chart
+
+## Description
+![#callout-1](img/utilization-bar-chart-callout-1.png)
+
+  1. **Chart Fill:**
+    - The fill color depends on the thresholds.
+    - When no thresholds exist the indicator is blue (#0088ce).
+    - When a threshold exists and the percentage has not surpassed any thresholds, the indicator is green (#3f9c35).
+    - When the utilization percentage has surpassed the warning threshold, but not the error threshold, the indicator is orange (#ec7a08)
```
Comment (LHinson) [inline pattern-library/data-visualization/utilization-bar-chart/design/design.md]: This is a bug on the site too. need a period after this sentence.

File: pattern-library/data-visualization/utilization-bar-chart/design/design.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    - When the utilization percentage has surpassed the error threshold, the indicator is is red (#cc0000).
+
+  1. **Background Fill:**
+    - The background color is grey (#d1d1d1)
```
Comment (LHinson) [inline pattern-library/data-visualization/utilization-bar-chart/design/design.md]: This is a bug on the site too. need a period after this sentence.

**Comment by serenamarie125** [comment]
@LHinson this should be good to go!  Let me know if there is anything else to resolve.

**Comment by LHinson** [comment]
bitmoji

**Pull request merged**